Lib/xml/etree/ElementTree.py

```diff
@@ -527,7 +527,10 @@ class ElementTree:
 
     """
     def __init__(self, element=None, file=None):
-        # assert element is None or iselement(element)
+        if element is not None and not iselement(element):
+            raise TypeError(f"element must be xml.etree.Element or "
```

Comment:
It is `xml.etree.ElementTree.Element`.

Look at `_assert_is_element()`. We need a similar error message.